package.json

```diff
@@ -13,33 +13,53 @@
     "dist"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.0.0-rc.1",
-    "@babel/core": "^7.0.0-rc.1",
-    "@babel/plugin-syntax-import-meta": "^7.0.0-rc.1",
-    "@babel/plugin-transform-modules-systemjs": "^7.0.0-rc.1",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-syntax-import-meta": "^7.2.0",
+    "@babel/plugin-transform-destructuring": "^7.2.0",
+    "@babel/plugin-transform-modules-systemjs": "^7.2.0",
+    "@babel/types": "^7.3.0",
+    "ajv": "^6.8.1",
     "bluebird": "^3.5.1",
     "esm": "^3.0.77",
+    "file-url": "^2.0.2",
+    "http-server": "^0.11.1",
     "mocha": "^5.2.0",
-    "rollup": "^0.64.1",
+    "rimraf": "^2.6.3",
+    "rollup": "^1.1.2",
+    "rollup-plugin-babel": "^4.3.2",
+    "rollup-plugin-commonjs": "^9.2.0",
+    "rollup-plugin-json": "^3.1.0",
+    "rollup-plugin-node-builtins": "^2.1.2",
+    "rollup-plugin-node-globals": "^1.4.0",
+    "rollup-plugin-node-resolve": "^4.0.0",
     "rollup-plugin-replace": "^2.0.0",
+    "source-map-support": "^0.5.10",
+    "strip-shebang": "^1.0.2",
     "terser": "^3.8.1",
+    "webpack": "^4.29.0",
+    "webpack-cli": "^3.2.1",
```

Comment:
This is a lot of dependencies!

Reply:
Webpack was the only way to get babel bundled for use in the browser.